reference/logical-replication.md

```diff
@@ -0,0 +1,316 @@
+```
+
+## [normal] Create table on Publisher and Subscriber
+
+This is a normal step.
+
+Create a normal user on Subscriber:
+
+Subscriber:
+
+```console
+% /usr/pgsql-{{ site.latest_postgresql_version }}/bin/createuser ${USER} -U postgres
+```
+
+Create a database on Publisher and Subscriber:
+
+Publisher and Subscriber:
+
+```console
+% /usr/pgsql-{{ site.latest_postgresql_version }}/bin/createdb --owner ${USER} blog -U postgres
+```
+
+Create a table in the created database.
+Connect to the created `blog` database:
+
+```console
+% /usr/pgsql-{{ site.latest_postgresql_version }}/bin/psql blog -U ${USER}
+```
+
+Create `entries` table:
+
+```sql
+CREATE TABLE entries (
+  title text,
+  body text
+);
+```
+
+## [special] Create a PGroonga index on Subscriber
+
+This is a PGroonga specific step.
+
+Install PGroonga to the database. It requires superuser privilege:
+
+Subscriber:
+
+```sql
+CREATE EXTENSION pgroonga;
+```
+
+Create a PGroonga index on Subscriber:
+
+Subscriber:
+
+```sql
+CREATE INDEX entries_full_text_search ON entries USING pgroonga (title, body);
+```
+
+Create a PUBLICATION on Publisher only:
+
+```sql
+CREATE PUBLICATION pub_srv1_blog FOR TABLE entries;
+```
+
+Create a SUBSCRIPTION on Subscriber only:
+
+```sql
+CREATE SUBSCRIPTION sub_srv2_blog CONNECTION 'dbname=blog hostaddr=172.16.0.2 port=5432 user=replicator password=passw0rd' PUBLICATION pub_srv1_blog;
+```
+
+## [normal] Insert data on Publisher only:
+
+Insert data to the created `entries` table:
+
+```sql
+INSERT INTO entries VALUES ('PGroonga', 'PGroonga is a PostgreSQL extension for fast full text search that supports all languages. It will help us.');
+INSERT INTO entries VALUES ('Groonga', 'Groonga is a full text search engine used by PGroonga. We did not know about it.');
+INSERT INTO entries VALUES ('PGroonga and replication', 'PGroonga 1.1.6 supports WAL based streaming replication. We should try it!');
+```
+
+Confirm replication:
+
+Publisher:
+
+```sql
+SELECT * FROM entries;
+ title    |                                  body
+ ---------+---------------------------------------------------------------------------
+ PGroonga | PGroonga is a PostgreSQL extension for fast full text search that supports all languages. It will help us.
+ Groonga  | Groonga is a full text search engine used by PGroonga. We did not know about it.
+ PGroonga and replication | PGroonga 1.1.6 supports WAL based streaming replication. We should try it!
+ (3 rows)
+```
+
+Subscriber:
+
+```sql
+SELECT * FROM entries;
+ title    |                                  body
+ ---------+---------------------------------------------------------------------------
+ PGroonga | PGroonga is a PostgreSQL extension for fast full text search that supports all languages. It will help us.
+ Groonga  | Groonga is a full text search engine used by PGroonga. We did not know about it.
+ PGroonga and replication | PGroonga 1.1.6 supports WAL based streaming replication. We should try it!
+ (3 rows)
+```
+
```

Comment:
Could you add a `SELECT` that uses the PGroonga index created only on subscriber?